subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/StaleOutputHistoryLossIntegrationTest.groovy

```diff
@@ -58,7 +58,6 @@ class StaleOutputHistoryLossIntegrationTest extends AbstractIntegrationSpec {
         result = runWithMostRecentFinalRelease(JAR_TASK_NAME)
 
         then:
-        javaProject.assertBuildTasksExecuted(result)
```

Comment:
The reason this worked before is because we ran the integration tests with `-l` to ensure we get the same output as before. Now that we are not running with lifecycle log level turned on the `OutputScrapingExecutionResult` would capture the task headers anymore. We'll need to revisit some of the integration tests later and probably implement a better way to capture the output than output scraping (see https://github.com/gradle/gradle/issues/1942).